src/index.html

```diff
@@ -3,14 +3,38 @@
   <head>
     <meta charset="UTF-8">
     <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="container">
+      <form
+        class="bigForm"
+        data-qa="big"
+        action="./"
+      >
+        <input
+          type="text"
+          class="input input--big"
+          data-qa="keypress"
+          placeholder="Try &quot;Los Angeles&quot;"
```

Comment:
class names should be appropriate to BEM, you have parent element - `form`, `form` is a B - bloc in this case, `input` is E - element, `input--big` is M - modifier, so `bigForm`, `input`, `input--big` are not suitable names in this case [BEM](https://mate-academy.github.io/style-guides/htmlcss.html#oocss-and-bem)
the same with smallForm